sway/desktop/xdg_shell.c

```diff
@@ -248,6 +253,24 @@ static void handle_request_fullscreen(struct wl_listener *listener, void *data)
 	transaction_commit_dirty();
 }
 
+static void handle_request_move(struct wl_listener *listener, void *data) {
+	struct sway_xdg_shell_view *xdg_shell_view =
+		wl_container_of(listener, xdg_shell_view, request_move);
+	struct sway_view *view = &xdg_shell_view->view;
+	struct wlr_xdg_toplevel_move_event *e = data;
+	struct sway_seat *seat = e->seat->seat->data;
+	seat_begin_move(seat, view->swayc);
+}
+
+static void handle_request_resize(struct wl_listener *listener, void *data) {
+	struct sway_xdg_shell_view *xdg_shell_view =
+		wl_container_of(listener, xdg_shell_view, request_resize);
+	struct sway_view *view = &xdg_shell_view->view;
+	struct wlr_xdg_toplevel_resize_event *e = data;
+	struct sway_seat *seat = e->seat->seat->data;
+	seat_begin_resize(seat, view->swayc, BTN_LEFT, e->edges);
```

Comment:
The client can request a resize when it's not the left button that is pressed - iirc mpv requests resize on right click.

Reply:
We can probably use the serial to know which button the client is talking about.